docs/rules/valid-typeof.md

```diff
@@ -28,6 +34,24 @@ typeof foo === baz
 typeof bar === typeof qux
 ```
 
+Examples of **incorrect** code with the `{ "requireStringLiterals": true }` option:
+
+```js
+typeof foo === undefined
+typeof bar == Object
+typeof baz === "strnig"
+typeof foo === 5
+typeof bar === typeof qux
+```
+
+Examples of **correct** code with the `{ "requireStringLiterals": true }` option:
+
+```js
+typeof foo === "undefined"
+typeof bar == "object"
+typeof baz === "string"
```

Comment:
There's `typeof baz === "strnig"`, which certainly signifies the intent (to catch typos), but it's easy to overlook. I'm +1 for adding  `typeof baz = "foo"` or `typeof baz = "someInvalidType"`as another example.